projects/plugins/jetpack/modules/shortcodes/presentations.php

```diff
@@ -106,7 +106,7 @@ public function add_scripts() {
 			}
 
 			foreach ( $GLOBALS['posts'] as $p ) {
-				if ( has_shortcode( $p->post_content, 'presentation' ) ) {
+				if ( isset( $p->post_content ) && has_shortcode( $p->post_content, 'presentation' ) ) {
```

Comment:
This is even better than the previous solution because it doesn't change the logic of the loop by introducing `continue`!